src/main/rst/03_webdriver.rst

```diff
@@ -356,6 +374,13 @@ Example of how to find an element that looks like this:
     @cheeses = $driver->find_elements('cheese', 'class');
 
 
+.. code-block:: javascript
+	
+	findElements(By.className("cheese")).then(function(cheeses){
```

Comment:
Less verbose if you use es6 arrow functions:

    driver.findElements(By.className("cheese"))
        .then(els => console.log(els.length));